cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Store.umc.tsx

```diff
@@ -14,7 +14,15 @@ const initialState = {
         // Cluster Settings advanced
         CONTROL_PLANE_NODES_COUNT: '1',
         WORKER_NODES_COUNT: '0',
-        CLUSTER_PROVIDER: '',
+        CLUSTER_PROVIDER: 'CALICO',
```

Comment:
I would suggest moving the Store to be in the umc area, and then using the field names for the initial state